### PR TITLE
[React] fix: improve Japanese localization

### PR DESCRIPTION
**Pull request by papparapa**
I am a native Japanese speaker and fixed some untranslated and unnatural parts of React UI in Japanese.

Changes are limited only to ja.json to replace text.

### CODE REVIEW

File: packages/react/common/lib/Localization/ja.json [hunk 1]
```diff
@@ -1,38 +1,38 @@
 {
   "sign_up": {
-    "email_label": "電子メールアドレス",
+    "email_label": "メールアドレス",
```
Comment (papparapa) [inline packages/react/common/lib/Localization/ja.json:3]: "電子メールアドレス" is not wrong, but "メールアドレス" or "Eメール" seems more natural.

File: packages/react/common/lib/Localization/ja.json [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
-    "email_label": "電子メールアドレス",
-    "password_label": "あなたのパスワード",
+    "email_label": "メールアドレス",
+    "password_label": "パスワード",
     "email_input_placeholder": "Your email address",
     "password_input_placeholder": "Your password",
     "button_label": "サインイン",
-    "social_provider_text": "に登録する",
-    "link_text": "Already have an account? Sign in"
+    "social_provider_text": "サインイン with",
```
Comment (papparapa) [inline packages/react/common/lib/Localization/ja.json:17]: The format `i18n[view]?.social_provider_text + ' ' +capitalize(provider)` is fixed, but there is no expression in Japanese that fits to this word order.
`に登録する Google` is clearly wrong word order in Japanese.
So I used more English-like expression.

File: packages/react/common/lib/Localization/ja.json [hunk 3]
```diff
@@ -1,38 +1,38 @@
...
     "button_label": "サインアップ",
-    "social_provider_text": "に登録する",
+    "social_provider_text": "サインイン with",
     "link_text": "アカウントをお持ちではありませんか？サインアップ"
   },
   "sign_in": {
-    "email_label": "電子メールアドレス",
-    "password_label": "あなたのパスワード",
+    "email_label": "メールアドレス",
+    "password_label": "パスワード",
```
Comment (papparapa) [inline packages/react/common/lib/Localization/ja.json:13]: "あなたのパスワード" seems a little redundant, "パスワード" is better.

**Comment by silentworks** [comment]
Thanks for this PR, we've moved all additional languages to a new directory in the root of this repository. This is because we no longer ship the additional languages with any of the packages, It is now up to the user to import the package they wish to use into their project by downloading the files from this repository.

**Pull request closed without merging**